ui/DesignSystem/Component/index.js

```diff
@@ -2,6 +2,7 @@ import AdditionalActionsDropdown from "./AdditionalActionsDropdown.svelte";
 import Copyable from "./Copyable.svelte";
 import HorizontalMenu from "./HorizontalMenu.svelte";
 import IdentityAvatar from "./IdentityAvatar.svelte";
+import NavigationButtons from "./Registration/NavigationButtons.svelte";
```

Comment:
If this becomes a generally exported component, it should have a more descriptive name. Navigation is a broad term. As it lives in our design system I'm curious to hear what @radicle-dev/product thinks this re-occurring element should be named.

Reply:
Also convinced that they don't need to live under `Registration`, it's a simple component which can be used in any form context with *cancle* and *submit* button.

Reply:
Waiting on @radicle-dev/product  for this one

Reply:
What does **NavigationButtons** do? Or what does it look like exactly? Is it directly related to the stepper? If so, maybe it could be called **StepperNavigationButtons**. If it's just the pink button and the "transparent" button, they should just be called **PrimaryButton**  and **TransparentButton**.

Reply:
@brandonhaslegs Thanks for the follow-up. It's the latter, so our navigation elements that we use in any form context. Would something like `FormNavigation` make sense?

Reply:
In most cases buttons are navigating forms, but they're definitely used in more contexts that just forms. For example are these `FormNavigation`?

![image](https://user-images.githubusercontent.com/4406983/80963557-73ff8280-8e0f-11ea-8d31-e74818298f15.png)
![image](https://user-images.githubusercontent.com/4406983/80963962-50890780-8e10-11ea-81a2-e3581f7ab3f1.png)
![image](https://user-images.githubusercontent.com/4406983/80964098-904fef00-8e10-11ea-9b3b-d675219bf0d3.png)


Maybe I'm misunderstanding the component we're talking about though. If it's truly just for forms, `FormNavigation` is fine with me.